server/routes/creatingLicences/handlers/initialMeetingName.ts

```diff
@@ -1,15 +1,18 @@
 import { Request, Response } from 'express'
+import LicenceService from '../../../services/licenceService'
 
 export default class InitialMeetingNameRoutes {
+  constructor(private readonly licenceService: LicenceService) {}
+
   GET = async (req: Request, res: Response): Promise<void> => {
-    const offender = {
-      name: 'Adam Balasaravika',
-    }
-    res.render('pages/create/initialMeetingPerson', { offender })
+    res.render('pages/create/initialMeetingPerson')
```

Comment:
This is the part where it may not be obvious that middleware is populating a licence based on pattern-matched URL and value.  Worth a comment in the places where we rely on the middleware doing its stuff?

Reply:
Since we will have this pattern almost everywhere, I have added an explanatory comment into the `index.ts` file which initialises the routes.